docs/content/noisy_simulation.md

```diff
@@ -0,0 +1,287 @@
+
+We consider a probabilistic combination $p_i$ of possible pure states $|\psi_i \rangle$. Thus, the system is described by a density matrix $\rho$ defined as follows:
+
+$$
+\rho = \sum_i p_i |\psi_i\rangle \langle \psi_i|
+$$
+
+The transformations of the density operator of an open quantum system interacting with its environment (noise) are represented by the super-operator $S: \rho \rightarrow S(\rho)$; super-operator can be written in the following form:
+
+$$
+S(\rho) = \sum_i K_i \rho K^{\dagger}_i
+$$
+
+Where $K_i$ are the Kraus operators, and satisfy the property $\sum_i K_i K^{\dagger}_i = \mathbb{I}$. As noise is the result of system interactions with its environment, it is therefore possible to simulate noisy quantum circuit with noise type gates.
+
+## Applicable noise types
+
+The noise models applicable in `Qadence` are classified into three types: digital, analog, and readout error. Digital and analog noise are applicable to `gates`, `blocks`, `circuits`, and `Models`. Readout error is applicable to `sample` and `expectation`.
+
+When dealing with programs involving digital operations, `Qadence` has interface to noise models implemented in `PyQTorch`. The following is a list of the supported types of noise. Detailed equation of noise options are available from [PyQTorch](https://pasqal-io.github.io/pyqtorch/latest/noise/).
+
+### Digital noise
+
+- BitFlip: flips between |0⟩ and |1⟩ with `error_probability`
+
+- PhaseFlip: flips the phase of a qubit by applying a Z gate with `error_probability`
+
+- Depolarizing: randomizes the state of a qubit by applying I, X, Y, or Z gates with equal `error_probability`
+
+- PauliChannel: applies one of the Pauli operators (X, Y, Z) to a qubit with specified `error_probabilities`
+
+- AmplitudeDamping: transitions qubit status from |1⟩ to |0⟩ with `damping_rate`
+
+- PhaseDamping: reduces coherence by randomizing the phase with `damping_rate`
+
+- GeneralizedAmplitudeDamping: extends amplitude damping; the first float is `error_probability` of amplitude damping, and second float is the `damping_rate`
+
+
+### Analog noise
+
+- Depolarizing: evolves to th maximally mixed state with `noise_probs`
+
+### Readout errors
+
+- Independent: all bits are corrupted with an equal `error_probability`
+- Correlated: apply `confusion_matrix` on error probabilities
+
+## NoiseHandler
+
+Noise models can be defined via the `NoiseHandler` type. It is a container of several noise instances which require to specify a `protocol` and a dictionary of `options` (or lists). The `protocol` field is to be instantiated from `NoiseProtocol` and `options` includes error-related information such as `error_probability`, `noise_probs`, and `seed`.
+
+```python exec="on" source="material-block" session="noise" result="json"
+from qadence import NoiseHandler
+from qadence.types import NoiseProtocol
+
+digital_noise = NoiseHandler(protocol=NoiseProtocol.DIGITAL.DEPOLARIZING, options={"error_probability": 0.1})
```

Comment:
I thought it'd be possible to instantiate noise from protocols only. If that's the case, I wouldn't jump directly to the handler to describe noise instantiations but rather use them in the previous sections and combine them in the handler here.